client/my-sites/plans/jetpack-plans/products-grid-i5/index.tsx

```diff
@@ -116,7 +121,7 @@ const ProductsGridI5: React.FC< ProductsGridProps > = ( {
 	}, [ onResize ] );
 
 	return (
-		<>
+		<Experiment name={ SWITCH_PLAN_SIDES_EXPERIMENT }>
```

Comment:
Required to record Tracks event, even though we don't use `DefaultVariation` and `Variation` here.